ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+HCP_CLI="/usr/bin/hcp"
+
+MCE=${MCE_VERSION:-""}
+CLUSTER_NAME=$(echo -n "${PROW_JOB_ID}"|sha256sum|cut -c-20)
+if [[ -n ${MCE} ]] ; then
+    CLUSTER_NAMESPACE_PREFIX=local-cluster
+else
+    CLUSTER_NAMESPACE_PREFIX=clusters
+fi
+#CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE_PREFIX}-${CLUSTER_NAME}
+
+if [[ -n ${MCE} ]] ; then
+  if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    # shellcheck source=/dev/null
+    source "${SHARED_DIR}/proxy-conf.sh"
+  fi
+
+  MCE_VERSION=$(oc get "$(oc get multiclusterengines -oname)" -ojsonpath="{.status.currentVersion}" | cut -c 1-3)
```

Comment:
I took it from here:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml#L24

but yes, if we decide to go with the `MCE_VERSION` being supplied from outside as in may other steps we can safely remove this.

Reply:
> but yes, if we decide to go with the MCE_VERSION being supplied from outside as in may other steps we can safely remove this.

If we're depending on the MCE_VERSION to exist in order for the MCE code paths to executed, then I think it should be safe to take the MCE_VERSION as it is passed in rather than detecting it.

Reply:
done